### ISSUE TITLE
Must-run CHPs should be dispatchable with the remainder of their capacity

### ISSUE THREAD

**Issue by ChaelKruip** [quintel/etengine#706]
One (known) shortcoming of the way we currently treat must-run CHPs in the ETM is that they are completely dominated by their production profile. This profile describes the demand for heat (not explicitly modeled in the ETM).
In reality, however, must-run CHPs will produce electricity whenever they can make money (just like other dispatchable plants). The problem of not modeling this behavior is that the ETM under-estimates the available dispatchable capacity and Merit will calculate electricity 'prices' that are too high.

In other words, the must-run profiles should describe when the plant runs **but not when it doesn't run**.

@antw can you make an estimate of the complexity of implementing this and how much time it would take a brilliant developer (you) to do that?

**Comment by ChaelKruip** [comment]
@antw can you make an estimate of how much time would be needed?

**Comment by michieldenhaan** [comment]
Closing, no longer applicable now that CHPs are fully dispatchable in Merit
